### PR TITLE
fill in service item param keys using pricing inputs spreadsheet

### PR DESCRIPTION
**Pull request by jacquelineIO**
## Description

Fill in database tables: `service_item_param_keys` and `service_params` following the [pricing inputs spreadsheet](https://docs.google.com/spreadsheets/d/1NRbxHmvaWV6aXQrxQ2LJhkc5tClAl3Eb1-MRxZ123Tw/edit#gid=0)

On lines 79 & 80 of the pricing inputs columns `E` and `F` are repeated and were renamed in the database tables to be `RateAreaNonStdOrigin` and `RateAreaNonStdDest` respectively.

## Reviewer Notes

For the database schema design refer to:
* https://dbdiagram.io/d/5ddd8e1eedf08a25543e6829
* https://docs.google.com/document/d/1f_FV8SeUAbmOb0zTvIidXhuHx00NzU4xpX0MiF2Ke-c/edit#heading=h.f1pr6eitdwyr

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

Run:
```sh
make db_dev_migrate
```

Verify using the spreadsheet of pricing inputs above. There are a lot, might be worth splitting the verification with another reviewer. 

The database count will be:
```sh
dev_db=# select count(*) from service_item_param_keys;
 count 
-------
    48
(1 row)

dev_db=# select count(*) from service_params;
 count 
-------
   280
(1 row)
```

## Code Review Verification Steps

* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1142?atlOrigin=eyJpIjoiNWNjOGExMzkxODI3NGM2NTg2NWU0Mjc0NzAwMzQwOTYiLCJwIjoiaiJ9) for this change
* [payment request database diagram](https://dbdiagram.io/d/5ddd8e1eedf08a25543e6829)
* [payment request database description](https://docs.google.com/document/d/1f_FV8SeUAbmOb0zTvIidXhuHx00NzU4xpX0MiF2Ke-c/edit#heading=h.f1pr6eitdwyr)
---
* [rate engine description](https://docs.google.com/document/d/1yLU-QBxuRTmZSzhZAm4LqAqxAnESVIxMGGeR9xHRZlM/edit#)

## Screenshots

n/a

### CODE REVIEW

File: migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+('6d1e8e68-4dcc-4ece-b96d-61ee3dfd67ea','PSI_PriceDomDestPrice', 'Payment Service Item Domestic Destination Price', 'DECIMAL', 'SYSTEM', now(), now()),
+('2e913e12-ac88-4425-bc89-a6bccb12734b','PSI_ShippingLinehaulIntlCO', 'Payment Service Item Intl CO Shipping & Linehaul', 'PaymentServiceItemUUID', 'SYSTEM', now(), now()),
+('8d3af988-0e2f-4ee0-9fbb-3df9235d01df','PSI_ShippingLinehaulIntlCOPrice', 'Payment Service Item Intl CO Shipping & Linehaul Price', 'DECIMAL', 'SYSTEM', now(), now()),
+('d5900ed8-f5e6-49a1-af4c-6958c1190500','PSI_ShippingLinehaulIntlOC', 'Intl OC Shipping & Linehaul', 'PaymentServiceItemUUID', 'SYSTEM', now(), now()),
+('c2813d4f-918a-450a-9b76-a1aa0a58b987','PSI_ShippingLinehaulIntlOCPrice', 'Intl OC Shipping & Linehaul', 'DECIMAL', 'SYSTEM', now(), now()),
+('afb75a59-8c88-4355-b55e-cc9eda672a55','PSI_ShippingLinehaulIntlOO', 'Intl OO Shipping & Linehaul Price', 'PaymentServiceItemUUID', 'SYSTEM', now(), now()),
+('421a4330-accc-44f4-9418-86fd5fc6195a','PSI_ShippingLinehaulIntlOOPrice', 'Intl OO Shipping & Linehaul', 'DECIMAL', 'SYSTEM', now(), now()),
```
Comment (reggieriser) [inline migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql]: The above four lines have descriptions that seem slightly off.  The first two have the same description (shouldn't one be marked "Price" like the one below it?).  The last two seem to have their descriptions swapped.  Also, some of the `PSI_` ones have `Payment Service Item` in their descriptions and others don't.

File: migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+('0891eda2-bbba-4f04-bf84-e5266b059f76','PSI_PackingDom', 'Domestic Packing', 'PaymentServiceItemUUID', 'SYSTEM', now(), now()),
+('ce6cd6db-d2af-4f0a-a17b-63bfb12f47a0','PSI_PackingDomPrice', 'Domestic Packing Price', 'DECIMAL', 'SYSTEM', now(), now()),
+('30bfc8e7-5219-4204-87e0-e96c46fcfe32','PSI_PackingHHGIntl', 'Intl HHG Packing', 'PaymentServiceItemUUID', 'SYSTEM', now(), now()),
+('5ec93806-b0a2-48e2-b7c7-fd1fce4eead0','PSI_PackingHHGIntlPrice', 'Intl HHG Packing Price', 'DECIMAL', 'SYSTEM', now(), now());
```
Comment (reggieriser) [inline migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql]: General nit: The given descriptions sometimes uppercase every word in the string, and sometimes doesn't (or even does both in the same string).  Not sure I have a preference of one or the other, but do prefer consistency.

File: migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql [hunk 3]
```diff
@@ -0,0 +1,433 @@
+ALTER TYPE service_item_param_type ADD VALUE 'PaymentServiceItemUUID' AFTER 'DECIMAL';
+
+INSERT INTO service_item_param_keys
+(id,key,description,type,origin,created_at,updated_at)
+VALUES
+('164050e3-e35b-480d-bf6e-ed2fab86f370','RequestedPickupDate', 'Customer requested pick up date', 'DATE', 'PRIME', now(), now()),
+('8c0c57e2-0acc-4319-9757-59608936a154','NumberDaysSIT', 'Customer requested pick up date', 'STRING', 'PRIME', now(), now()),
```
[the remark this answers is not in the document]
Comment (jacquelineIO) [reply migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql]: updated

File: migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+('5a3cf5bb-2bbf-4af6-8966-143d52d8f94b','SITScheduleDest', 'Dest SIT Schedule', 'STRING', 'SYSTEM', now(), now()),
+('cd37b2a6-ac7d-4c93-a148-ca67f7f67cff','ServicesScheduleOrigin', 'Origin Services Schedule', 'STRING', 'SYSTEM', now(), now()),
+('33751a6a-0a20-4ee2-9e46-ece2c1e997ba','ServicesScheduleDest', 'Dest Services Schedule', 'STRING', 'SYSTEM', now(), now()),
+('86f9b2b6-6315-409f-a563-e91998a5043f','PriceAreaOrigin', 'Origin Price Area', 'STRING', 'SYSTEM', now(), now()),
+('07e3ff12-0717-4661-a4e1-dca325da0fcc','PriceAreaDest', 'Dest Price Area', 'STRING', 'SYSTEM', now(), now()),
+('6d44624c-b91b-4226-8fcd-98046e2f433d','PriceAreaIntlOrigin', 'Intl Origin Price Area', 'STRING', 'PRIME', now(), now()),
+('4736f489-dfda-4df1-a303-8c434a120d5d','PriceAreaIntlDest', 'Intl Dest Price Area', 'STRING', 'PRIME', now(), now()),
+('0f564bdd-153c-44af-97ae-117b62339c02','RateAreaNonStdOrigin', 'Non Std Rate Area', 'STRING', 'PRIME', now(), now()),
+('73060cd4-15fd-4d6d-9d38-9f362c7da7af','RateAreaNonStdDest', 'Non Std Rate Area', 'STRING', 'PRIME', now(), now()),
```
[the remark this answers is not in the document]
Comment (jacquelineIO) [reply migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql]: Yea, that changed later and I didn't updated the description, thanks!

File: migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql (hunk 1 above)
Comment (jacquelineIO) [reply migrations/20200107000006_payment_request_add_service_item_param_keys.up.sql]: updated

**Pull request merged**